### PR TITLE
I can haz locations

### PR DESCRIPTION
**Pull request by Rumel**
It works now. I need people to pull down this branch and make sure their database doesn't freak out though.

### CODE REVIEW

File: Radabite/ConnectionStrings.config [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ﻿<connectionStrings>
-  <add name="DefaultConnection" connectionString="Data Source=(LocalDb)\v11.0;Integrated Security=SSPI" providerName="System.Data.SqlClient" />
+  <add name="DefaultConnection" connectionString="Data Source=(LocalDb)\v11.0;Integrated Security=SSPI;database=radabite;" providerName="System.Data.SqlClient" />
```
Comment (clottman) [inline Radabite/ConnectionStrings.config]: Insignificant, but can we make the database name Radabite with a capital R?
Just for the prettiness.

Comment (Rumel) [inline Radabite/ConnectionStrings.config]: I suppose

File: Radabite/Global.asax.cs [hunk 2]
```diff
@@ -41,6 +41,15 @@ protected void Application_Start()
             BundleConfig.RegisterBundles(BundleTable.Bundles);
             AuthConfig.RegisterAuth();
 
+
+            using(var db = new Radabite.Backend.Database.Db())
+            {
+                if(db.Database.Exists())
+                {
+                    db.Database.Initialize(true);
```
Comment (clottman) [inline Radabite/Global.asax.cs]: Is this initializing the connection? Or the actual db like making the tables?

**Comment by clottman** [comment]
:ship:

**Comment by NathanDeMaria** [comment]
:shipit:

**Comment by clottman** [comment]
^^ Cassey Lottman likes this :shipit:

**Pull request merged**